akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -1199,6 +1199,7 @@ trait LoggingAdapter {
   protected def notifyError(message: String): Unit
   protected def notifyError(cause: Throwable, message: String): Unit
   protected def notifyWarning(message: String): Unit
+  protected def notifyWarning(cause: Throwable, message: String): Unit
```

Comment:
So the default implementation protecting custom LoggingAdapters would be:
```suggestion
  protected def notifyWarning(cause: Throwable, message: String): Unit = notifyWarning(message)
```

Not entirely sure MiMa will understand this is safe though. We might still need the filter.